<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphics</class>
 <widget class="QDialog" name="Graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: gray;</string>
  </property>
  <widget class="QPushButton" name="pushButton_draw">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>440</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_max">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_min">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_y">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_y">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Range X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Range Y</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>571</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="func">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
qproperty-alignment: 'AlignVCenter';</string>
   </property>
   <property name="text">
    <string>func</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="butt_step">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
